verification/curator-service/ui/src/components/LinelistTable.tsx

```diff
@@ -260,7 +258,11 @@ class LinelistTable extends React.Component<Props, LinelistTableState> {
                                 title: 'Age',
                                 field: 'age',
                                 filtering: false,
-                                type: 'numeric',
+                                editable: 'never',
+                                render: (rowData) =>
+                                    rowData.age[0] === rowData.age[1]
+                                        ? rowData.age[0]
+                                        : `[${rowData.age[0]},${rowData.age[1]})`,
```

Comment:
The mocks have "0-1" and is more readable than this syntax IMO.

Reply:
ok changed to xx-yy